src/routes/InstancesPage/CreateInstanceModal.js

```diff
@@ -173,13 +186,26 @@ function CreateInstanceModal({
         <FormGroup label="Cloud region" isRequired fieldId="region">
           <SelectSingle
             id="region"
+            isDisabled={isFetchingRegions}
             value={formValues.region}
             handleSelect={onCloudRegionSelect}
           >
-            {regionOptions.map((region) => {
+            {/*
+              Adding an empty & disabled option as default, otherwise the default item will be the first item returned
+              from the api, which might be inconsistent.
+            */}
+            <SelectOption
+              key="placeholder"
+              value=""
+              isPlaceholder
+              isDisabled={true}
+            >
+              &nbsp;
```

Comment:
Rather than a space, let's present a user-friendly prompt as placeholder.

Something simple and straightforward, like,
```suggestion
              Choose a region
```